src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2ScenarioHelper.cs

```diff
@@ -25,11 +25,13 @@ namespace Sql2.Tests.ScenarioTests
 {
     class Sql2ScenarioHelper
     {
-        
+
         /// <summary>
         /// The region in which the tests will create their needed resources
         /// </summary>
-        private static string TestEnvironmentRegion = "Australia East";
+        private static string TestEnvironmentRegion = "North Europe";
```

Comment:
Please update this so you are not simply using hard-coded regions.  These regiosn may or may not be available depending on which environment or test cluster is being targeted.